v2/openapi/fb-provider-collateral-api.yaml

```diff
@@ -747,22 +725,37 @@ components:
         - **Rejected** - The transaction has been rejected and the account has not been credited
     CollateralDepositTransaction:
       type: object
-      required: [ fireblocksAssetId, collateralTxId ]
+      required: [ collateralTxId ]
       properties:
         collateralTxId:
           type: string
           description: >
             A unique identifier of the transaction to track.
             This field will contain information to help the provider poll the status of the transaction from Fireblocks.
           example: "0.8e4cfce8-0182-4c6d-b9dd-a291c105e1d2.0.5a814998-ec0f-4f1c-92bf-fb5f7dc09ea2"
-        fireblocksAssetId:
-          type: string
-          description: "ID of the Fireblocks asset"
         amount:
           $ref: '#/x-schemas/PositiveAmount'
         status:
           $ref: '#/components/schemas/CollateralDepositTransactionStatus'
 
+    CollateralDepositTransactions:
+      allOf:
+        - type: object
+          required:
+            - id
+          properties:
+            id:
+              type: string
```

Comment:
The issue here is that we are using CollateralDepositTransaction object inside Register a collateral deposit transaction request body, what will make us create the id param instead of the customer.

Reply:
In any case the name should not be in plural – this is a single object.

Reply:
done